tests/cases/test_raw_node.rs

```diff
@@ -59,21 +59,22 @@ fn conf_change(t: ConfChangeType, node_id: u64) -> ConfChange {
     cc
 }
 
-fn new_ready(
```

Comment:
Maybe we can `#[derive(PartialEq)]` instead?

Reply:
Ah, nevermind. :)

Reply:
emm.. do you mean to add the constructor or just keep it as it is?